pkgs/games/prismlauncher/default.nix

```diff
@@ -0,0 +1,91 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  jdk8,
+  jdk,
+  zlib,
+  file,
+  wrapQtAppsHook,
+  xorg,
+  libpulseaudio,
+  qtbase,
+  libGL,
+  quazip,
+  glfw,
+  openal,
+  extra-cmake-modules,
+  tomlplusplus,
+  ghc_filesystem,
+  msaClientID ? "",
+  jdks ? [jdk jdk8],
+}: let
+  libnbtplusplus = fetchFromGitHub {
+    owner = "ljfa-ag";
+    repo = "libnbtplusplus";
+    rev = "v2.5";
+    sha256 = "sha256-4PHKPbWIWAy9ym25M4SnllTkrqnR+g/0AOqyMBZcdX4=";
+  };
+in
+  stdenv.mkDerivation {
+    pname = "prismlauncher";
+    version = "unstable-2022-10-18";
+
+    src = fetchFromGitHub {
+      owner = "PrismLauncher";
+      repo = "PrismLauncher";
+      rev = "243600b75babc636bdd1ac0dc8fd5fa4a2db1a0c";
+      sha256 = "sha256-cDwa6NeZew+MubtRmUViLA8gUaoH3AMxD9PdQcgSgSU=";
+    };
+
+    nativeBuildInputs = [extra-cmake-modules tomlplusplus ghc_filesystem cmake file jdk wrapQtAppsHook];
+    buildInputs = [qtbase zlib quazip];
```

Comment:
```suggestion
    nativeBuildInputs = [ extra-cmake-modules ghc_filesystem cmake file jdk wrapQtAppsHook ];
    buildInputs = [ qtbase zlib quazip tomlplusplus ];
```

Reply:
tomlplusplus is a header-only library and there for a build-time dependency

Reply:
Which style-guide to use is already mentioned up here: https://github.com/NixOS/nixpkgs/pull/196624#discussion_r998543473

alejandra doesn't format like that, if we switch to nixpkgs-fmt I have no doubt that the formatting will change

Reply:
Ahhh, gotcha, I didn't realize that it had moved into buildInputs, my bad

Reply:
I am not aware of any rule that moves header only libraries to nativeBuildInputs. As far as I am aware it should go to buildInputs. and aljandra is incomaptible with nixpkgs.